circle.yml

```diff
@@ -478,7 +478,13 @@ jobs:
       - install-dependencies
       - install-ios-packaging-dependencies
       - run:
-          name: Trigger external deploy steps
+          name: Trigger documentation deploy steps
+          command: |
+            export VERSION_TAG=${CIRCLE_TAG}
+            export DOCS_REPO="mapbox-gl-native-ios"
```

Comment:
I think this should be `ios-sdk`. I tried this locally and accidentally created a docs PR in this repo.